packages/manager/modules/navbar/src/index.scss

```diff
@@ -3,7 +3,7 @@
 
   background-color: $p-000-white;
   box-shadow: $base-box-shadow-primary;
-  z-index: 100;
+  z-index: 1032;
```

Comment:
I'm not sure we should play with `z-index`, it oftens look like we put a random number so it works

Reply:
It looks like a random number, but it's set compared to the navbar and the backdrop.

navbar: 1032
sidebar: 1031
backdrop: 1030

Why we've started so high, I don't know. I suppose it was based on the value from Bootstrap.

Reply:
So we can maybe create a var which expose the z-index of the navbar?

Reply:
Or maybe directly use the values of the UI kit :P
https://github.com/ovh/ovh-ui-kit/blob/v2.x/packages/oui-navbar/_variables.less#L11

I'll update it, it should be clearer if all if regrouped in a single place ;)

Reply:
I've didn't see that the file was a `SCSS` one. So it'll be difficult to use UI-kit variables for now.

Reply:
@AxelPeter I think you can import the variables from v4, I don't know if that is of any help with the `z-index`

Reply:
IMO, I think that importing a new package as big as the UI Kit, just to use some variables, is a bit to much (I mean here with v2 + v4). Of course we lose the centralized side of the thing by doing like that.

To see a little farther than just the `z-index`. It should be done as a whole with the way we manage the main page layout of all the managers. But it's something that will be planned very soon :)